src/main/java/fr/insee/queen/api/controller/SurveyUnitController.java

```diff
@@ -1,10 +1,7 @@
 package fr.insee.queen.api.controller;
 
 import java.io.IOException;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.List;
-import java.util.Optional;
+import java.util.*;
```

Comment:
don't import the Entire package but only class used

Reply:
fixed...IDE issue when > 5 imports of the same package